modin/pandas/test/test_concat.py

```diff
@@ -45,6 +45,17 @@ def generate_dfs():
     return df, df2
 
 
+def generate_multiindex_dfs():
+    def generate_multiindex(index):
+        return pandas.MultiIndex.from_tuples(
+            [("a", x) for x in index.values], names=["name1", "name2"]
+        )
+
+    df1, df2 = generate_dfs()
+    df1.columns, df2.columns = map(generate_multiindex, [df1.columns, df2.columns])
+    return df1, df2
+
+
```

Comment:
It's better to move all `generate_*` functions to `test/utils.py`.

Reply:
done